cross/chromaprint/Makefile

```diff
@@ -9,13 +9,13 @@ HOMEPAGE = https://acoustid.org/chromaprint
 COMMENT  = Chromaprint is the core component of the AcoustID project. It\'s a client-side library that implements a custom algorithm for extracting fingerprints from any audio source.
 LICENSE  = LGPL2.1+
 
-OPTIONAL_DEPENDS = cross/ffmpeg4
+OPTIONAL_DEPENDS = cross/ffmpeg$(FFMPEG_VERSION)
 
 # compiler too old
 UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(OLD_PPC_ARCHS)
 
 ifneq ($(wildcard $(FFMPEG_DIR)),)
-CMAKE_RPATH = /var/packages/ffmpeg$(FFMPEG_VERSION)/target/lib
+CMAKE_RPATH = /var/packages/ffmpeg$(subst 4,,$(FFMPEG_VERSION))/target/lib
```

Comment:
@th0ma7 I suppose without this fix the chromaprint package is not able to find the ffmpeg libs at runtime.

Reply:
You are totally right, good catch.